ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -259,15 +264,15 @@ tests:
     workflow: openshift-e2e-azure-upi
```

Comment:
`e2e-azure-upi` had `skip_if_only_changed` added in your previous patch b438968355fa498e0e7af29cc426f266dc8ace19, but you probably didn't intend for that to run on most PRs either.

Reply:
I was hoping to run this when only upi files are changed but it ran for all tests for some reason. I did fix it again.